app.js

```diff
@@ -76,6 +80,8 @@ if (app.get('env') == 'development') {
   app.use(errorHandler());
 }
 
+const c = "vuln c"
+
 var token = 'SECRET_TOKEN_f8ed84e8f41e4146403dd4a6bbcea5e418d23a9';
 console.log('token: ' + token);
```

Comment:
### <sub><img src="https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-medium.svg" width="20"/></sub>&nbsp; Use of Hardcoded Credentials

Do not hardcode passwords in code. Found hardcoded password used in password_token.

<h6>
<a href="https://cwe.mitre.org/data/definitions/798.html">CWE-798</a> | <a href="https://cwe.mitre.org/data/definitions/259.html">CWE-259</a> | <a href="https://docs.snyk.io/manage-risk/prioritize-issues-for-fixing/priority-score">Priority score 562</a> | Line 2</h6>

<details>

<summary><b>Data flow: 10 steps</b></summary>

Step 1: `SOURCE` https://github.com/trailblazing-dx/tiago_app.dev.snyk_github_app_nodejs-goof/blob/2187ce28013c2862d62c3f7fea128055deeb459a/app.js#L47

Step 2: [app.js#L47](https://github.com/trailblazing-dx/tiago_app.dev.snyk_github_app_nodejs-goof/blob/2187ce28013c2862d62c3f7fea128055deeb459a/app.js#L47)

Step 3: [app.js#L47](https://github.com/trailblazing-dx/tiago_app.dev.snyk_github_app_nodejs-goof/blob/2187ce28013c2862d62c3f7fea128055deeb459a/app.js#L47)

Step 3: [app.js#L47](https://github.com/trailblazing-dx/tiago_app.dev.snyk_github_app_nodejs-goof/blob/2187ce28013c2862d62c3f7fea128055deeb459a/app.js#L47)

Step 3: [app.js#L47](https://github.com/trailblazing-dx/tiago_app.dev.snyk_github_app_nodejs-goof/blob/2187ce28013c2862d62c3f7fea128055deeb459a/app.js#L47)

Step 3: [app.js#L47](https://github.com/trailblazing-dx/tiago_app.dev.snyk_github_app_nodejs-goof/blob/2187ce28013c2862d62c3f7fea128055deeb459a/app.js#L47)

Step 3: [app.js#L47](https://github.com/trailblazing-dx/tiago_app.dev.snyk_github_app_nodejs-goof/blob/2187ce28013c2862d62c3f7fea128055deeb459a/app.js#L47)

Step 3: [app.js#L47](https://github.com/trailblazing-dx/tiago_app.dev.snyk_github_app_nodejs-goof/blob/2187ce28013c2862d62c3f7fea128055deeb459a/app.js#L47)

Step 3: [app.js#L47](https://github.com/trailblazing-dx/tiago_app.dev.snyk_github_app_nodejs-goof/blob/2187ce28013c2862d62c3f7fea128055deeb459a/app.js#L47)

Step 4: `SINK`

https://github.com/trailblazing-dx/tiago_app.dev.snyk_github_app_nodejs-goof/blob/2187ce28013c2862d62c3f7fea128055deeb459a/app.js#L83

</details>